Source/WSTagsField.swift

```diff
@@ -401,6 +416,15 @@ open class WSTagsField: UIScrollView {
         repositionViews()
     }
 
+    open func setRemovable(tags: [String], removable: Bool = false) {
+        assert(tagViews.count > 0, "There are no tagViews. Did you call this method after adding tags?")
```

Comment:
Please try to find another way avoiding the use of `asserts`. This is quite unsafe in runtime.